lib/concurrent-ruby/concurrent/executor/java_executor_service.rb

```diff
@@ -46,6 +46,7 @@ def shutdown
       def kill
         synchronize do
           @executor.shutdownNow
+          ok = @executor.awaitTermination(60, java.util.concurrent.TimeUnit::SECONDS) until ok
```

Comment:
Could you reuse `wait_for_termination` here?
Also I think this line should be moved the `synchronize`, it's typically problematic to do any blocking call in `synchronize`.